frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -182,18 +203,18 @@ const AdminShoutouts: React.FC = () => {
     });
   }
 
-  const displayShoutouts = allShoutouts
-    ?.filter((shoutout) => {
-      const shoutoutDate = new Date(shoutout.timestamp);
-      return (
-        (view === 'ALL' ||
-          (view === 'PRESENT' && !shoutout.hidden) ||
-          (view === 'HIDDEN' && shoutout.hidden)) &&
-        shoutoutDate >= earlyDate &&
-        shoutoutDate <= lastDate
-      );
-    })
-    .sort((a, b) => a.timestamp - b.timestamp);
+  // const displayShoutouts = allShoutouts
+  // ?.filter((shoutout) => {
+  //   const shoutoutDate = new Date(shoutout.timestamp);
+  //   return (
+  //     (view === 'ALL' ||
+  //       (view === 'PRESENT' && !shoutout.hidden) ||
+  //       (view === 'HIDDEN' && shoutout.hidden)) &&
+  //     shoutoutDate >= earlyDate &&
+  //     shoutoutDate <= lastDate
+  //   );
+  // })
+  // .sort((a, b) => a.timestamp - b.timestamp);
```

Comment:
This is what should be updating `displayShoutouts`. Since `allShoutouts` is part of the component state, whenever `setAllShoutouts` is called to update the state, it causes a re-render of the component. This code that's commented should run every render of the component to derive `displayShoutouts` from `allShoutouts`

Reply:
ahh i fixed it, was missing the async keyword💔

Reply:
i made these changes, however, the displayShoutouts list contents still doesnt update upon rerender when allShououts changes

Reply:
Hmm good catch... I think this is likely an issue with time zones on the server. We had a similar issue with deadlines for dev portfolio before. I'm not sure if that fix was also made everywhere else timestamps are used.

Either way I think this problem isn't within the scope of this PR. @andrew032011 should be something to look into in the future.